### DISCUSSION TITLE
using builtin_profiles nnvgpu profile, output video vertical direction is reversed

### DISCUSSION THREAD

**Discussion by Sobev** [time4tea/gopro-dashboard-overlay#216] [Q&A]
<img width="990" alt="image" src="https://github.com/user-attachments/assets/221ba0d5-6bc5-4c92-9ebe-ba67e5563326">
Does anyone have the same problem?

**Comment by bphilli7** [comment]
My solution, which is just one possible solution to this problem, was to create a user defined profile in the json file in my home directory that copied the parameters of the built-in profile exactly but had this line for "filter"
`"filter": "[0:v]rotate=PI:c=none:ow=rotw(0):oh=roth(0)[rotate0];[rotate0][1:v]overlay",`

I named it something like "nnvgpu-rotated". Ffmpeg will rotate the gopro footage 180 degrees when it prepares the overlayed footage.
